lib/controllers/meeting.js

```diff
@@ -0,0 +1,24 @@
+const Meeting = require("../models/meeting");
+
+module.exports = {
+  display: (req, res) => {
+    const { owner, participants, date } = req.body;
+
+    try {
+      res.json({ owner, participants, date });
+      res.sendStatus(200);
+    } catch (err) {
+      res.status(500).send("Something went wrong!");
+      console.error("This be the err: ", err);
+    }
+  },
+  create: async (req, res) => {
```

Comment:
```suggestion
  create: async (req, res) => {
    const { owner, participants, date } = req.body;
```
We have to unpack those from the request body before we can reference them in `Meeting.create`.

Reply:
Added